packages/react-dom-bindings/src/client/ReactFiberConfigDOM.js

```diff
@@ -2601,26 +2614,21 @@ function preloadStylesheet(
 ) {
   preloadPropsMap.set(key, preloadProps);
 
-  if (!ownerDocument.querySelector(getStylesheetSelectorFromKey(key))) {
```

Comment:
Why is this check removed? If there is an existing style sheet why would we preload it?

Reply:
We only call this function in one place and I changed it to gate the call on the instance not existing so we don't need to duplicate the check here